src/toil_scripts/adam_pipeline/adam_preprocessing.py

```diff
@@ -36,11 +36,12 @@
 
 import yaml
 from toil.job import Job
```

Comment:
Small nit: can you move `import yaml` into the earlier block of imports. This way, all of the `from toil.*` imports will be in a single block.